spring-boot/admin/src/main/resources/templates/admin/fragments/itemDeleter.html

```diff
@@ -0,0 +1,307 @@
+<!DOCTYPE html>
```

Comment:
I have concerns about that suggestion. The main purpose of this class provide JS functionality to delete an item. The dialog is just an extra feature containing a confirmation dialog to make sure this purpose shall be fulfilled. Your suggestions do not mention the actual purpose which I find confusing.

Reply:
I get your point, however this is an HTML file, where I usually expect mainly GUI elements rather than functionality. This is why "deleter" seems confusing as it sounds like a functional modules.

What about `itemDeletionAndDialogFragments` or `itemDeleterAndConfirmationDialogs` or ...?

Reply:
Ah okay, now I get your point and agree. I also liked your suggestions and prefer `itemDeletionAndDialogFragments` as the user will not be surprised to find an "extendValidity" dialog.